autoload/nodeinspect/config.vim

```diff
@@ -34,20 +34,32 @@ function! s:ReplaceMacros(str)
 	return replaced
 endfunction
 
+function s:getReadableConfigFile(path)
+	let configFilePath = a:path . '/' . s:configFileName
+	if filereadable(configFilePath)
+		return configFilePath
+	endif
+	let legacyConfigFilePath = a:path . '/' . s:legacyConfigFileName
```

Comment:
I believe s:legacyConfigFileName should be declared at the top (see line 1 let s:configFileName = ...)

Reply:
Huh, yeah it super should be, and is in my environment, I don't know how I missed adding that to the commit. I just pushed up another commit to add them 👍